compose/material3/adaptive/adaptive-layout/build.gradle

```diff
@@ -149,7 +149,7 @@ tasks.register("updateTranslations", UpdateTranslationsTask.class) {
     ]
     targetDirectory = project.file("src/skikoMain/kotlin/androidx/compose/material3/adaptive/l10n")
     targetPackageName = "androidx.compose.material3.adaptive.l10n"
-    kotlinStringsPackageName = "androidx.compose.material3.adaptive.layout"
+    kotlinStringsPackageName = "androidx.compose.material3.adaptive.layout.internal"
```

Comment:
As far as I understand, this task still works in `jb-main`?

If so, we need to call it every merge (from [the process description](https://docs.google.com/document/d/16ieNsWjj-zXecry0aGx8xEo85HBR6DWdL6bW9veKX1c/edit?tab=t.kiy4p8panypk#heading=h.xgtsktjpxtaq)), not once closer to the release. Did you call it for the merges?